docs/source/notebooks_and_ipython/kedro_and_notebooks.md

```diff
@@ -84,6 +84,23 @@ catalog.list()
 
 When you run the cell:
 
+```ipython
+['example_iris_data',
+ 'parameters',
+ 'params:example_test_data_ratio',
+ 'params:example_num_train_iter',
+ 'params:example_learning_rate'
+]
+```
+
+#### Search datasets with regex
+If you do not remember the exact name of a dataset, you can provide regular expression to search datasets.
+```ipython
+catalog.list("example*")
+```
+
+When you run the cell:
+
 ```ipython
 ['example_iris_data',
  'parameters',
```

Comment:
Why does `catalog.list("example*")` give `parameters`?

Reply:
That's a typo! 😅 I was making this example up since it's still using the `pandas-iris` starter, I do test it with the `spaceflights` one and it works as expected.

Reply:
I don't understand - shouldn't we correct the output?

Reply:
Agree with @astrojuanlu that you probably forgot to remove `parameters` from the result below. Didn't get the explanation as well 🙈

Reply:
😅 the commit never push through on GitPod that's why it's not making any sense